### PR TITLE
Doc334 uninstallation instruct

### PR DESCRIPTION
**Pull request by StarfallProjects**
PR for: https://github.com/EventStore/documentation/issues/334

### CODE REVIEW

File: server/uninstalling.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+This document describes how to uninstall Event Store. The instructions depend on which installation method you used. For different installation methods, refer to [Getting started - Step 1](~/getting-started/index.md). These instructions cover how to uninstall Event Store, but not how to remove dependencies such as the .NET framework or Mono.
+
+### [Windows](#tab/tabid-1)
+
+### Chocolatey
+
+If you installed Event Store with Chocolatey, you can uninstall with:
+
+```
```
Comment (ChrisChinchilla) [inline server/uninstalling.md]: @StarfallProjects I am a fan of using full syntax highlightig

Comment (ChrisChinchilla) [inline server/uninstalling.md]:
```suggestion
```posh
```

I think for powershell

File: server/uninstalling.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+* Deleting the `EventStore-OSS-Win-*` directory.
+* Removing the directory from your PATH.
+
+### [Linux](#tab/tabid-2)
+
+### Pre-built packages
+
+If you installed one of the [pre-built packages for Debian based systems](https://packagecloud.io/EventStore/EventStore-OSS), you can remove it with:
+
+```
```
Comment (ChrisChinchilla) [inline server/uninstalling.md]:
```suggestion
```shell
```

File: server/uninstalling.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+If you built Event Store from source, remove it by deleting the directory containing the source and build, and manually removing any environment variables.
+
+### [macOS](#tab/tabid-macos)
+
+<!--NOTE: the following needs to be tested. I am basing it off some googling -->
+
+If you installed Event Store using Homebrew, you can remove it with one of the following:
+
+```
```
Comment (ChrisChinchilla) [inline server/uninstalling.md]: No, this isn't correct, it's much simpler.

```shell
brew cask uninstall eventstore
```

Is it. Actually the cask formula needs more work, but this is the command

**Comment by StarfallProjects** [comment]
Fixed the syntax highlighting

File: server/uninstalling.md (hunk 3 above)
Comment (StarfallProjects) [reply server/uninstalling.md]: Does that remove everything though? I bow to your superior macOS knowledge, but my googling suggested you needed the --force flag to properly clean everything up (make sure all versions are removed)

Comment (ChrisChinchilla) [reply server/uninstalling.md]: I have never used the --force option unless it's a dependency of something else and you want to force it. Let's go without it for now and I'll do some full testing later.

Comment (StarfallProjects) [reply server/uninstalling.md]: Ok, have removed the zap and --force stuff

**Pull request merged**